test/ios_xctestrun_runner_unit_test.sh

```diff
@@ -50,6 +50,44 @@ ios_xctestrun_runner(
     reuse_simulator = False,
 )
 
+genrule(
+  name = "pre_action_gen",
+  executable = True,
+  outs = ["pre_action.bash"],
+  cmd = """
+echo 'echo PRE-ACTION' > \$@
```

Comment:
Could we update this to use some of the env so we can test they get forwarded

Reply:
done!